Dockerfile

```diff
@@ -1,15 +1,16 @@
 FROM registry.access.redhat.com/ubi8:latest
 
 RUN dnf install -y --nodocs git python3-pip hostname dmidecode pciutils procps-ng && dnf clean all
-RUN pip3 install --no-cache-dir  ansible elasticsearch elasticsearch-dsl openshift kubernetes redis
 RUN mkdir -p /backpack && chmod 770 /backpack
 ENV HOME /backpack
 WORKDIR /backpack
+COPY stockpile-wrapper/stockpile-wrapper.py stockpile-wrapper.py
+COPY stockpile-wrapper/requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
```

Comment:
I think you forgot to git add the requirements file ;-)

Reply:
fixed!